.github/actions/issue-labeler/regexes-for-issue-labels.yaml

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+# Adds <issue_type> label if issue body contains certain words (when based on the issue template)
+
+# regular expressions for labeling priority of the issue is listed in this block
+Critical:
+  - '\*{2}Priority:\*{2} ?([Cc]ritical|[Uu]rgent)'
+High:
+  - '\*{2}Priority:\*{2} ?[Hh]igh'
+Medium:
+  - '\*{2}Priority:\*{2} ?[Mm]edium'
+Low:
+  - '\*{2}Priority:\*{2} ?[Ll]ow'
+
+# regular expressions for labeling issue type is listed in this block
+BUG:
+  - "bug"
+FEATURE:
+  - "feature( request)?"
+# ENHANCEMENT:
+# IDEA:
+# QUESTION:
+# SUPPORT:
+
+# regular expressions for labeling stakeholders for an issue is listed in this block
+cms:
+  - '\*{2}Stakeholders:\*{2} ?(CMS|Cms|cms)'
+factoryops:
+  - '\*{2}Stakeholders:\*{2} ?([Ff]actory[Oo]ps)'
+fermilab:
+  - '\*{2}Stakeholders:\*{2} ?([Ff]ermilab)'
+fife:
+  - '\*{2}Stakeholders:\*{2} ?(FIFE|Fife|fife)'
+igwn:
+  - '\*{2}Stakeholders:\*{2} ?(IGWN|Igwn|igwn)'
+osg:
+  - '\*{2}Stakeholders:\*{2} ?(OSG|Osg|osg)'
```

Comment:
Will this recognize the stakeholder also if multiple are listed?
If not you can change the regex or ask to list one per line (repeating "-  Stakeholders:")

Reply:
The original regexes (shown in the code snipper above) was not capable of recognizing multiple stakeholders. As per the suggestion, I have modified the list of regexes to handle multiple stakeholders scenario and have also updated the issue templates requesting users to list one stakeholder per line.